shared/bridges/cctp/cosmos_bridge_client.go

```diff
@@ -230,11 +230,7 @@ func (c *CosmosBridgeClient) FillOrder(ctx context.Context, order db.Order, gate
 		},
 	}
 	if order.Data.Valid {
-		data, err := hex.DecodeString(order.Data.String)
-		if err != nil {
-			return "", "", nil, fmt.Errorf("decoding hex order data to string: %w", err)
-		}
-		fillOrderMsg.FillOrder.Order.Data = data
+		fillOrderMsg.FillOrder.Order.Data = order.Data.String
```

Comment:
ah so the gateway wanted the hex string? tbh I wasn't sure how the gateway expected this data so ty for this fix